### PR TITLE
feat: Add magic login link

### PR DESCRIPTION
**Pull request by didrikmunther**
## Describe your changes

Fixes: #882 

## Checklist before requesting review

- [x] Feature/fix PRs should add one atomic (as small as possible) feature or fix.
- [x] The code compiles and all the tests pass.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204650493731358

### CODE REVIEW

**Comment by didrikmunther** [comment]
Icon shown in CRM, triggering a fetch call for a magic link:

Screenshot 2023-05-25 at 21 25 13

Popup screen with magic link, shown to sales:

Screenshot 2023-05-25 at 21 24 41

Error page (for person using already used magic link):

Screenshot 2023-05-25 at 21 23 52

Log in page (for person using a magic link, needing to press the log in button to be redirected to the /register page):

Screenshot 2023-05-25 at 21 24 30

File: api/views.py [hunk 1]
```diff
@@ -325,6 +326,19 @@ def partners(request):
     return JsonResponse(data, safe=False)
 
 
+def create_magic_link(request):
```
Comment (didrikmunther) [inline api/views.py:329]: Could use some documentation on this function

Comment (didrikmunther) [inline api/views.py:329]: It is not specified how long a magic link should last. I think default is 5 minutes, which might be too little.

**Review by Hampfh — DISMISSED** [review]
Cool!

File: api/views.py (hunk 1 above)
Comment (didrikmunther) [reply api/views.py:329]: Also there is a security issue, as anyone can create a magic link here.

**Pull request merged**